tensorflow/lite/micro/kernels/add_test.cc

```diff
@@ -27,15 +27,16 @@ namespace testing {
 namespace {
 
 // Shapes and values for mixed broadcast tests.
-constexpr int broadcast_output_dims_count = 36;
-constexpr int broadcast_num_shapes = 4;
+constexpr int kBroadcastOutputDimsCount = 36;
```

Comment:
Thanks for making these changes as you go along!

Reply:
Fixing lint issues can be very fulfilling! 🧹 ✨